src/std/db/postgresql-driver.ss

```diff
@@ -23,9 +23,19 @@
         postgresql-close-statement!
         postgresql-exec!
         postgresql-query!
+        postgresql-simple-query!
         postgresql-continue!
         postgresql-reset!
         postgresql-close!
+        postgresql-socket
```

Comment:
do we need to export all these?

Reply:
They are all used by the client in postgresql.ss with the exception of the socket, and are also quite useful when developing a GUI/Shell for postgresql itself rather than an app that uses it as a database.

Since for the most part anyone who is using the driver is in fact doing something low level there is little reason to keep anything hidden. Thoughts?

Reply:
ok fair enough, lets keep them.